src/app/api/join/route.ts

```
@@ -0,0 +1,11 @@
+import { NextResponse } from 'next/server';
+
+import type { NextRequest } from 'next/server';
+
+export const POST = async (request: NextRequest) => {
+  const {} = await request.json();
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)